dot-gen/src/main/java/com/wipro/ats/bdre/ldg/GetDotForTable.java

```diff
@@ -0,0 +1,132 @@
+                logger.info("------------------------inside src check");
+                relationDot.append("\n" + lineageRelation.getDotString() + "\n");                                            //search for target table name in the DOT string
+                logger.info("------------------" + relationDot);
+                targetTableNode.append("\n" + getLineageNodeByColName.getTableDotFromNodeId(lineageRelation.getLineageNodeByTargetNodeId(), srcTableNodeList.get(0)) + "\n");
+                logger.info("-------------------------" + targetTableNode);
+                if(targetTableNode.equals("same-nodes")) {
+                    targetTableNode.append("\n" + getLineageNodeByColName.getTableDotFromNodeId(lineageRelation.getLineageNodeBySrcNodeId(), srcTableNodeList.get(0)) + "\n");
+                }
+                logger.info("------------------------------relation DOT: \n" + relationDot);
+             }
+
+            //this should check whether the nodes are target -- ??
+            lineageRelationListForTargetNodes = getNodeIdForLineageRelation.getWhenTargetNode(lineageNode.getNodeId());
+            for(LineageRelation lineageRelation:lineageRelationListForTargetNodes) {
+                //if (lineageRelationList.size() != 0) {
+                logger.info("------------------------inside target check");
+                relationDot.append("\n" + lineageRelation.getDotString() + "\n");                                            //search for target table name in the DOT string
+                logger.info("------------------" + relationDot);
+                targetTableNode.append("\n" + getLineageNodeByColName.getTableDotFromNodeId(lineageRelation.getLineageNodeBySrcNodeId(), srcTableNodeList.get(0)) + "\n");
+                logger.info("-------------------------" + targetTableNode);
+                if(targetTableNode.equals("same-nodes")) {
+                    targetTableNode.append("\n" + getLineageNodeByColName.getTableDotFromNodeId(lineageRelation.getLineageNodeByTargetNodeId(), srcTableNodeList.get(0)) + "\n");
+                }
+                logger.info("-------------------------------relation DOT: \n" + relationDot);
+            }
+
+        }
+        dotString = lineageMain.generateLineageDot(relationDot.toString(), srcTableNodeList.get(0).getDotString(), targetTableNode.toString());             //call lineage main to create the DOT
+        return dotString;
+    }
+
+    //fn() to get DOT through only Table name
+    public String dotGeneratorWithTable(String args[]) {
+        if(args.length !=1) {
+            logger.info("One argument required: please Give Table name");
+            System.exit(-1);
+        }
+        GetLineageNodeByColName getLineageNodeByColName = new GetLineageNodeByColName();;
+        LineageMain lineageMain = new LineageMain();
+        String srcTableName = args[0];
+        logger.info("Generating DOT for the given Table Name: " + args[0]);
+        String targetTableName = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "targetTableName" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)